core/dbt/artifacts/resources/v1/catalog.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional
+
+from dbt.adapters.catalogs import CatalogIntegrationConfig
+from dbt_common.dataclass_schema import dbtClassMixin
+
+
+@dataclass
+class CoreCatalogIntegrationConfig(CatalogIntegrationConfig):
+    name: str
```

Comment:
This protocol now has a `catalog_name` in addition to `name`. `name` should be the name of the catalog and `catalog_name` should be the name of the catalog integration. In other words, I think `name` here and `name` on `Catalog` below are the same thing.

```suggestion
    name: str
    catalog_name: str
```